compiler/jsgen.nim

```diff
@@ -370,7 +370,7 @@ type
 
 const # magic checked op; magic unchecked op;
   jsMagics: TMagicOps = [
-    ["addInt", ""], # AddI
+    mAddI: ["addInt", ""],
```

Comment:
how about using https://github.com/nim-lang/Nim/pull/11424 all the way, (ie replacing each comment by a litteral index as you started doing) => would guarantee it does the right thing at CT, especially as some magics are being removed

but this also could be done in later PR's

Reply:
I already did it in my other PR: https://github.com/nim-lang/Nim/pull/13536/files